malfeasance/handler.go

```diff
@@ -44,6 +44,7 @@ const (
 	InvalidActivation MalfeasanceType = iota + 10
 	InvalidBallot
 	InvalidHareMsg
+	DoubleMarry = MalfeasanceType(wire.DoubleMarry)
```

Comment:
Lets sync on this tomorrow and try to integrate - the double marry type isn't defined here. I outlined the basic structure here with the first malfeasance proofs: https://github.com/spacemeshos/go-spacemesh/pull/6043/